shared/src/packages/surveys/libs/types/survey-request-dto.types.legacy.ts

```diff
@@ -0,0 +1,6 @@
+type SurveyRequestDto = {
```

Comment:
what do you mean by legacy? if it's not used anymore, remove it

Reply:
done